tests/test_config.py

```diff
@@ -164,8 +164,7 @@ def test_experiment_defined_parameters(self):
             python.read_nonblocking(10000)
             python.setecho(False)
             python.sendline('from dallinger.experiment_server import experiment_server')
-            python.sendline('from dallinger.config import get_config')
-            python.sendline('config = get_config()')
+            python.sendline('config = experiment_server._config()')
```

Comment:
It looks like this is just a wrapper for `get_config` that checks some internal state, what's the reason for this switch? Were there errors?

Reply:
Yes, I forget the details, but the original version was relying on some side effect that no longer happens on import.